src/commands/synthetics/reporters/junit.ts

```diff
@@ -392,8 +499,43 @@ export class JUnitReporter implements Reporter {
           {$: {name: 'type', value: test.type}},
         ].filter((prop) => prop.$.value),
       },
-      testcase: [],
+      skipped: [],
       warning: [],
     }
   }
+
+  private getTestCaseStats(result: Result): TestCaseStats {
+    let stepsStats: TestCaseStats[] = []
+    if ('stepDetails' in result.result) {
+      // It's a browser test.
+      stepsStats = result.result.stepDetails
+        .map((step) => {
+          if (!step.subTestStepDetails) {
+            return [step]
+          }
+
+          return [step, ...step.subTestStepDetails]
+        })
+        .reduce((acc, val) => acc.concat(val), [])
```

Comment:
I was about to suggest using `flatMap` (or `map(...).flatten()`) as it's been available since Node 11, but I just saw we are still on Node 10.21 on this repository 😞

Reply:
Yes, but it's on purpose because of Azure DevOps sadly.